gate-web/src/main/java/com/opsmx/spinnaker/gate/audit/AuthenticationAuditListener.java

```diff
@@ -60,6 +61,7 @@ public void onApplicationEvent(AbstractAuthenticationEvent event) {
       } else if (!event.getAuthentication().isAuthenticated()
           && event instanceof AbstractAuthenticationFailureEvent) {
         log.debug("publishEvent AbstractAuthenticationFailureEvent");
+        log.info("Login : User '"+event.getAuthentication().getName()+"' login failed due to incorrect credentials.");
```

Comment:
```suggestion
        log.info("User : {} login failed due to incorrect credentials.", event.getAuthentication().getName());
```
Can we change the log statement like this ?